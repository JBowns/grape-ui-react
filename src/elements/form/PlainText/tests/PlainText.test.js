import React from 'react';
import renderer from 'react-test-renderer';
import { ThemeProvider } from 'styled-components';
import { PlainText } from '..';
import 'jest-styled-components';

describe('PlainText Component base', () => {
  it('should return a PlainText object', () => {
    const component = renderer.create(<ThemeProvider theme={{}}><PlainText>Lorem Ipsum</PlainText></ThemeProvider>);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
