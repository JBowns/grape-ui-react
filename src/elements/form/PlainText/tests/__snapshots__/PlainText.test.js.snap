// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlainText Component base should return a PlainText object 1`] = `
.c0 {
  background-color: transparent;
  color: inherit;
  font-family: -apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Helvetica,Arial,sans-serif,"Apple Color Emoji","Segoe UI Emoji","Segoe UI Symbol";
  font-size: inherit;
  font-weight: inherit;
  -webkit-letter-spacing: inherit;
  -moz-letter-spacing: inherit;
  -ms-letter-spacing: inherit;
  letter-spacing: inherit;
  line-height: 1.5;
  padding-bottom: 1rem;
  padding-left: 1rem;
  padding-right: 1rem;
  padding-top: 1rem;
  text-align: inherit;
  -webkit-text-decoration: inherit;
  text-decoration: inherit;
}

<div
  className="c0"
>
  
</div>
`;
